docs/WindowsSubsystemForLinux.md

```diff
@@ -0,0 +1,94 @@
+# Running on Windows Subsystem for Linux
+
+Contact: Andrew Pardoe, @AndrewPardoe on GitHub
```

Comment:
What about `[https://github.com/AndrewPardoe](@AndrewPardoe)` so you get an acutal link? (GitHub should be doing this already, but...)

Reply:
Yes, will edit that.